app/code/pagetype/HomePage.php

```diff
@@ -67,7 +67,7 @@ public function getCMSFields()
 
         $fields->addFieldToTab('Root.Banner', $bannerImage = UploadField::create(
             'BannerImage',
-            'Banner image'
+            'Banner Image (only supports jpg, jpeg, png filetypes)'
```

Comment:
you could use the `setDescription()` function to insert the file types and also ask james on what text it should say in regards to what image dimensions should be uploaded